components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/beans/AuthorizationRequest.java

```diff
@@ -0,0 +1,52 @@
+package org.wso2.carbon.device.mgt.jaxrs.beans;
```

Comment:
fixed it with https://github.com/wso2/carbon-device-mgt/pull/426/commits/ec3dbd37ddc3c4695b6bb19b8e3eb3323b2aa1a8